testnet/stacks-node/src/config.rs

```diff
@@ -957,12 +957,18 @@ impl Config {
         }
 
         let miners_contract_id = boot_code_id(MINERS_NAME, is_mainnet);
-        if node.miner
-            && burnchain.mode == "nakamoto-neon"
-            && !node.stacker_dbs.contains(&miners_contract_id)
-        {
-            debug!("A miner must subscribe to the {miners_contract_id} stacker db contract. Forcibly subscribing...");
-            node.stacker_dbs.push(miners_contract_id);
+        let signers_contract_id = boot_code_id(SIGNERS_NAME, is_mainnet);
+        println!("miners_contract_id: {}", miners_contract_id);
```

Comment:
Please don't use `println!` or `eprintln!`.  Instead, use the logging facilities